lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -31,8 +31,31 @@ def colorized_message_lines(colorizer=::RSpec::Core::Formatters::ConsoleCodes)
           end
         end
 
-        def formatted_backtrace
-          backtrace_formatter.format_backtrace(exception_backtrace, example.metadata)
+        def formatted_backtrace(exception = @exception)
+          lines = backtrace_formatter.format_backtrace((exception.backtrace || []), example.metadata)
+          potential_lines = []
+
+          while exception.respond_to?(:cause) && exception.cause
```

Comment:
In the gemfile add `, branch:  :exception_cause_supported` to the `rspec-support` repo, you'll have to pull it out of the loop to do so. Do it in a temporary commit that can be dropped.